### PR TITLE
Feat/organize plots

### PR DESCRIPTION
**Pull request by johpiip**
This is a terrible mess of miscellaneous commits, sorry!
- Reorganized scripts into better (?) folder structure
- Tweaks plots
- Adds new maps
- Updates CO2 levels and factors

### CODE REVIEW

File: scripts/figures/single/map_twocenters.R [hunk 1]
```diff
@@ -0,0 +1,270 @@
+# -*- coding: utf-8-unix -*-
```
Comment (zptro) [inline scripts/figures/single/map_twocenters.R:1]: Everything in this file is done four times in a row. Could transform this into a function that is called four times?

File: scripts/basemap/centers_uusimaa-2050.R [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+library(sf)
+
+zones <- readr::read_rds(here::here("results", "zones.rds"))
+
+centers <- here::here("data", "Uusimaa-kaava-2050_Pisteet.gpkg") %>%
+  sf::read_sf() %>%
+  sf::st_transform(3879) %>%
+  dplyr::filter(grepl("^Keskustatoimintojen alue", kuvaus)) %>%
+  dplyr::filter(sf::st_intersects(., sf::st_combine(zones), sparse = FALSE)) %>%
+  sf::st_join(zones, join = st_within)
```
Comment (zptro) [inline scripts/basemap/centers_uusimaa-2050.R:13]: Nice automatization!

**Review by zptro — CHANGES_REQUESTED** [review]
I do not understand everything, but in general the code looks "healthy".

File: scripts/figures/single/map_twocenters.R (hunk 1 above)
Comment (johpiip) [reply scripts/figures/single/map_twocenters.R:1]: Fixed in https://github.com/HSLdevcom/mal-effect-calculations/pull/30/commits/fa6225c174c8fc3ba779b41f2e06a2e075187ea2 https://github.com/HSLdevcom/mal-effect-calculations/pull/30/commits/328c40b67dcf4d52a94c2a2b7a3b8413550d1f1e https://github.com/HSLdevcom/mal-effect-calculations/pull/30/commits/2d3a56c30375252ed7521db2a257dd048183b65c

**Review by zptro — APPROVED** [review]
Nice work!

**Pull request merged**